docs/how-to/jupyter-notebook/connect-mlflow.rst

```diff
@@ -0,0 +1,74 @@
+Connect from Notebook to MLFlow
+===============================
+
+This guide provides instructions on how to integrate MLFlow with your Jupyter Notebook in the Data Science Stack (DSS) environment for tracking experiments.
+
+Overview
+--------
+
+MLFlow is a platform for managing the end-to-end machine learning life cycle. It includes tracking experiments, packaging code into reproducible runs, and sharing and deploying models. DSS environments are pre-configured to interact with an MLFlow server through the `MLFLOW_TRACKING_URI` environment variable set in each notebook.
+
+Prerequisites
+-------------
+
+Before you begin, ensure the following:
+
+- You have a Jupyter Notebook running in the DSS environment.
```

Comment:
can we swap the word `running` with `active` here, so that we are consistent with the terminology of the `dss list` command?